python/servo/visual_studio.py

```diff
@@ -164,13 +164,14 @@ def find_windows_sdk_installation_path() -> str:
 
     # This module must be imported here, because other platforms also
     # load this file and the module is platform-specific.
-    import winreg
-
-    # This is based on the advice from
-    # https://stackoverflow.com/questions/35119223/how-to-programmatically-detect-and-locate-the-windows-10-sdk
-    key_path = r"SOFTWARE\Wow6432Node\Microsoft\Microsoft SDKs\Windows\v10.0"
-    try:
-        with winreg.OpenKeyEx(winreg.HKEY_LOCAL_MACHINE, key_path) as key:
-            return str(winreg.QueryValueEx(key, "InstallationFolder")[0])
-    except FileNotFoundError:
-        raise Exception(f"Couldn't find Windows SDK installation path in registry at path ({key_path})")
+    if sys.platform == "win32":
```

Comment:
I'm really surprised this is necessary. What was the issue? Maybe an assertion is enough?